components/snackbar.css

```diff
@@ -0,0 +1,37 @@
+
+
+.body{
+    background-color: lightgray;
+   
+  }
+  
+  .snack-container {position : fixed;
+  top : 85vh;
+  left : 40vw;
+  right: 40vw;
+     font-family : "comic sans ms";
+    background-color : darkgray;
+    display : flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    width : 25rem;
+    padding-top : 0.625rem;
+    padding-bottom : 0.625rem;
+    border-radius : 0.3125rem;
+    border : 0.125rem solid black;
+  }
+  .snack-footer{ font-family : "comic sans ms";
```

Comment:
you've already given the same font family to the parent container, so i believe reapplying the font style here is meaningless.